pyproject.toml

```diff
@@ -0,0 +1,9 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "setuptools_scm[toml]>=6.4",
+    "wheel",
```

Comment:
`wheel` is auto-injected via the setuptools' PEP 517 build back-end. Also, there's a bit missing that allows tagged `pip install https://github//...git/archive.tar.gz` to work.
```suggestion
    # Essentials
    "setuptools >= 42",

    # Plugins
    "setuptools_scm[toml] >= 6.4",
    "setuptools_scm_git_archive >= 1.1",
```
For `setuptools_scm_git_archive` to work, don't forget to add `.git_archival.txt` containing:
```txt
ref-names: $Format:%D$
```
and `.gitattributes` with:
```gitattributes
# Force LF line endings for text files
* text=auto eol=lf

# Needed for setuptools-scm-git-archive
.git_archival.txt  export-subst
```

Reply:
moved the PEP 517 migration (without setuptools-scm) to #746, explicit wheel dependency is removed there.

Reply:
implemented in #748